master/static/migrations/20210922170815_update-model-registry-get-apis.up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE public.modelsv2 (
+	id integer NOT NULL,
+	name character varying UNIQUE NOT NULL,
+	description character varying,
+	metadata jsonb,
+	creation_time timestamp with time zone NOT NULL,
+	last_updated_time timestamp with time zone,
+    num_versions integer DEFAULT 0 NOT NULL,
```

Comment:
Yes, but the tradeoff is storage (one column) vs. database query anytime someone asks for a model - does that tradeoff make sense? If so, I'm happy to do it via a query since it should just be a join and count.

Reply:
I'd strongly prefer letting postgres do its job and implement this via join-count. I see denormalization as a premature optimization.

Reply:
Sure - will change!